DESCRIPTION

```diff
@@ -38,10 +38,13 @@ Suggests:
     mlr3learners,
     mlr3measures,
     mlr3pipelines,
+    mlr3proba,
     praznik,
     rpart,
     testthat (>= 3.0.0),
     withr
+Remotes:
+    mlr-org/mlr3proba
```

Comment:
```suggestion
```

Reply:
I had done the day before yesterday Sebastian :) just ran out of battery at the airport